server/channels/app/bot_test.go

```diff
@@ -112,6 +112,82 @@ func TestCreateBot(t *testing.T) {
 	})
 }
 
+func TestEnsureBot(t *testing.T) {
+	t.Run("ensure bot should pass if already exist bot user", func(t *testing.T) {
+		th := Setup(t).InitBasic()
+		defer th.TearDown()
+		pluginId := "pluginId"
+
+		PluginErr := th.App.SetPluginKey(pluginId, "key", []byte("test"))
+		assert.Nil(t, PluginErr)
```

Comment:
In the Mattermost code base, two types of errors are used: The standard `error` interface and the custom `*model.AppError` struct. To avoid confusing the return (See https://go.dev/doc/faq#nil_error for more details) the naming pattern is the following: `error`s are named `err` and `*model.AppError` is named `appErr`. While the code isn't always consistent with that naming schema, it should be used for newly added tests like the one in this PR.

Could you please update the code to take that naming schema into account?

Reply:
Thank you very much for explanation!